assets/src/components/vehicleIcon.tsx

```diff
@@ -175,7 +175,7 @@ const TooltipContent = React.memo(
       <br />
       <b>Adherence:</b> {scheduleAdherenceLabel}
       <br />
-      <b>Operator:</b> {operatorDetails}
+      <b>Operator:</b> <span className="fs-mask">{operatorDetails}</span>
```

Comment:
**question:** Have you tested the visual appearance just to double-check that putting the operator details inside of a `<span>` doesn't change anything? I doubt it would but it's probably worth looking into just to be sure.

Reply:
Yeah it looked the same!